### ISSUE TITLE
Invalid Path when Cloning

### ISSUE THREAD

**Issue by Juaaanits** [Virtual-Protocol/agent-contributions-library#111]
![Screenshot (1462)](https://github.com/user-attachments/assets/dc194cd6-9826-4a68-b201-d32a4091ae27)

**Comment by armlynobinguar** [comment]
Hi @juannits, 

Download the zip file instead of your forked repository. Unzip it once you downloaded it and open it on your vscode.

<img width="1340" alt="Screenshot 2024-09-02 at 2 58 22 PM" src="https://github.com/user-attachments/assets/64ca1448-bdec-41df-b162-729f757fdffc">

**Comment by armlynobinguar** [comment]
Once you downloaded the zip file. You'll be able to have a copy and click that file to unzip

<img width="383" alt="Screenshot 2024-09-02 at 2 59 53 PM" src="https://github.com/user-attachments/assets/eacab40e-e5cb-4f92-8634-9766ba351ffc">

**Comment by armlynobinguar** [comment]
Once the file is downloaded and unzip the file, open your vscode and click this. 

<img width="1490" alt="Screenshot 2024-09-02 at 3 07 55 PM" src="https://github.com/user-attachments/assets/b02e61cb-1a30-42ba-8cef-0538133a8c91">

<img width="1490" alt="Screenshot 2024-09-02 at 3 08 02 PM" src="https://github.com/user-attachments/assets/b909d244-ce01-4ee1-ac15-941d52daf4c4">

The look should be like this

**Comment by Juaaanits** [comment]
Fixed na po, Ma'am. Then ipapublish ko po ba sa GitHub?
![image](https://github.com/user-attachments/assets/2b11c1cd-b6e9-4f54-baab-0a169be42305)

**Comment by armlynobinguar** [comment]
Yes please, visit your forked repository and add click the contribute

**Comment by Juaaanits** [comment]
![image](https://github.com/user-attachments/assets/85639bc8-25f0-4460-a5ad-3c1d4394c887)
![image](https://github.com/user-attachments/assets/dbb7d298-b704-4e2d-95d9-9358e02c6d28)

**Comment by armlynobinguar** [comment]
Can you try to do git init?

**Comment by Juaaanits** [comment]
![image](https://github.com/user-attachments/assets/f92198c2-9711-4e3b-8e89-510213dd8f10)

**Comment by armlynobinguar** [comment]
Try to add the files again, delete it and add again

**Comment by armlynobinguar** [comment]
Can you also try doing git status?

**Comment by Juaaanits** [comment]
![image](https://github.com/user-attachments/assets/22d14823-a60e-4956-8fc0-8531fc3acf88)

**Comment by armlynobinguar** [comment]
Weird, it should be showing that you've added files. Delete the file and retry again

**Comment by armlynobinguar** [comment]
do the git checkout main again

**Comment by Juaaanits** [comment]
done po Ma'am
![image](https://github.com/user-attachments/assets/6717610e-a47a-4890-b34b-3fa37ba6f667)

**Comment by Juaaanits** [comment]
Ulitin ko na lang po. Thank you po!
Nagawa ko na rin po ito last time eh

**Comment by armlynobinguar** [comment]
Sure sure, redo it langs.

**Comment by Juaaanits** [comment]
Hindi po ba talaga magwowork through git clone po Ma'am tulad ng dati?

**Comment by armlynobinguar** [comment]
You can still do the git clone but it will take time since there's large files or many folders na din sa repo.

**Comment by Juaaanits** [comment]
![image](https://github.com/user-attachments/assets/9ed85905-2413-4963-8a33-a505a87a381f)
![image](https://github.com/user-attachments/assets/b9f33df7-c89f-41df-927c-f20a0f05d0ef)
Hello po ulit, Ma'am. Ano po next dito?

**Comment by armlynobinguar** [comment]
Click the compare and request
